doc/bn.tex

```diff
@@ -875,46 +880,31 @@ \subsection{Long Constants}
 
 \subsection{Long Constants - platform dependant}
 
-\index{mp\_set\_long}
+\index{mp\_set\_ulong}
 \begin{alltt}
-int mp_set_long (mp_int * a, unsigned long b);
+void mp_set_slong (mp_int * a, long b);
+void mp_set_ulong (mp_int * a, unsigned long b);
 \end{alltt}
 
 This will assign the value of the platform-dependent sized variable $b$ to the mp\_int $a$.
 
-To get the ``unsigned long'' copy of an mp\_int the following function can be used.
-
-\index{mp\_get\_long}
-\begin{alltt}
-unsigned long mp_get_long (mp_int * a);
-\end{alltt}
-
-This will return the least significant bits of the mp\_int $a$ that fit into an ``unsigned long''.
-
-\subsection{Long Long Constants}
-
-\index{mp\_set\_long\_long}
-\begin{alltt}
-int mp_set_long_long (mp_int * a, unsigned long long b);
-\end{alltt}
-
-This will assign the value of the 64-bit variable $b$ to the mp\_int $a$.
-
-To get the ``unsigned long long'' copy of an mp\_int the following function can be used.
+To retrieve the value, the following function can be used.
```

Comment:
"the following functions", plural, you made two out of The One That Was Before.
Didn't see it the first time, sorry.

Reply:
fixed